end-to-end/cypress/integration/dataset-search-tutorial.spec.js

```diff
@@ -0,0 +1,80 @@
+import defaultSearch from '../fixtures/dataset-search/default';
+import limitedDataTypesSearch from '../fixtures/dataset-search/heart-only-5-or-less-data-types';
+
+
+function traverseSteps(steps){
+  cy.wrap(steps).each((step, i) => {
+    cy.findByText(step.content).should('exist');
+    cy.findByText(`${step.title} (${i+1}/${steps.length})`).should('exist');
+    if (i < steps.length - 1) {
+      cy.findByRole('button', { name: 'Next' }).click();
+    } else {
+      cy.findByRole('button', { name: 'Finish Tutorial' }).click();
+    }
+    
+  });
+};
+
+function assertTileView(){
+  cy.findByTestId('tile-view-toggle-button').should('have.attr', 'aria-pressed', 'true');
+  cy.findByTestId('search-results-tiles').should('exist');
+  cy.findByTestId('search-tiles-sort-button').should('exist');
```

Comment:
```suggestion
  cy.findByTestId('search-tiles-sort-button').should('be.visible');
```
 Not sure if that's valid, but could it exist without being visible?